client/curriculum/html/LinksAndButtons.tsx

```diff
@@ -0,0 +1,882 @@
+              nou)
+            </strong>
+          </p>
+        </section>
+        <section>
+          <LessonHeading as="h3" id="bune-practici">
+            Bune practici
+          </LessonHeading>
+          <p>
+            Când folosim valoarea _blank, e recomandat să adăugăm un nou atribut
+            pentru securitate și anume :
+            {' '}
+            <strong className="formatted">
+              rel cu valorile ”noopener noreferrer”.
+            </strong>
+            {' '}
+            Atributul
+            {' '}
+            <strong className="formatted">rel</strong>
+            {' '}
+            poate avea mai
+            multe valori, dar le vom prezenta aici doar pe cele 2 recomandate a
+            se folosi împreuna cu atributul target=”_blank”.
+          </p>
+          <Highlight
+            className="my-5"
+            language="html"
+            code={`
+<a href="http://frontend.ro" target="_blank" rel="noopener noreferrer">
+  Link-ul se deschide într-un nou tab
+</a>
+            `}
+          />
+          <LessonFigure
+            withBorder
+            src="/images/lessons/links-and-buttons/rel.png"
+            alt="Atributul rel cu valorile noopener și noreferrer"
+            demo="/demo/html/noopener-noreferrer-rel"
+          />
+          <ul className="with--bullets">
+            <li>
+              <strong className="formatted">rel="noopener"</strong>
+              : blochează
+              accesul tabului nou la pagina ce conține link-ul
+            </li>
+            <li>
+              <strong className="formatted">rel="noreferrer"</strong>
+              : împiedică
+              alte site-uri sau servicii de urmărire (de exemplu, Google
+              Analytics) să identifice pagina ca sursă a link-ului pe care s-a
+              dat click.
+            </li>
+          </ul>
+        </section>
+        <section>
+          <LessonHeading as="h3" id="intrebari-bonus">
+            Întrebări BONUS
+          </LessonHeading>
+          <p>
+            În continuare vom raspunde la câteva întrebări pe care noi ca și
+            developeri ni le putem adresa pe măsură ce codăm și ne întâlnim cu
+            diverse situații:
+          </p>
+          <blockquote>
+            Ar trebui să folosim atributul
+            {' '}
+            <strong className="formatted">title</strong>
+            {' '}
+            împreuna cu tag-ul a?
+          </blockquote>
+          <p>
+            Hmmm..probabil că nu. În momentul în care mergem cu mouse-ul în
+            dreptul elementului (facem
+            {' '}
+            <i>hover</i>
+            ) ce are ca și atribut title
+            vom vedea un pop-up cu textul pe care l-am scris.
+          </p>
+          <Highlight
+            className="my-5"
+            language="html"
+            code={`
+<a title="Nu ar trebui să fiu aici" href="/">
+  Un simplu exemplu
+</a>`}
+          />
+          <LessonFigure
```

Comment:
Replace with a more relevant picture in which we actually show how "title" looks when hovered. Get rid of the demo page, and inline this link.